### DISCUSSION TITLE
Question about DAMMIF online version

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [biosaxs-com/atsas-community#784] [Ab initio shape determination]
On 2012-03-24 15:56:03, user spark wrote:

Dear ATSAS-Team,

I noticed your web interface changed slightly.
Why does DAMMIF now allow to change / choose the search volume, please?
There is the option based on D_Max and unlimited.
Why does unlimited make sense, please?

I am declined to use D_Max for my search volume as I don't expect my particle to be bigger.

Thank you for explanation.

Kind regards
S.

**Comment by franke-biosaxs** [comment]
On 2012-03-26 13:29:09, user spark wrote:

Sorry, there are now a couple more questions after the run has finished.
It looks like the current DAMMIF online version uses automatically damaver.
That is nice.

I just don't understand what the clustering is about. My pdb files were divided into groups, one composed of 3 pdb files and the other contains 15 files.
Could you please explain what this means, where I could read more information about it, and why I have suddenly two references rather one, please?

Thank you.
Spark

**Comment by franke-biosaxs** [comment]
On 2012-03-26 15:45:02, user SaxsMax wrote:

In online Dammif interface bead modelling
is followed by clustering of the models (using damclust).
It is an approach somewhat alternative to damaver one,
namely it assumes that there are no outliers all the generated models
may exist and the program simply groups them into clusters
based on similarity. Within each cluster damaver routine
is performed to get the representative.
More details can be found here:
"New developments in the ATSAS program package for small-angle scattering data analysis". J. Appl. Cryst. 45, 342-250
(check [http://www.embl-hamburg.de/biosaxs/publications.html](http://www.embl-hamburg.de/biosaxs/publications.html) for the link to pdf)

**Comment by franke-biosaxs** [comment]
On 2012-03-26 17:18:48, user franke wrote:


spark wrote
> There is the option based on D_Max and unlimited.
> Why does unlimited make sense, please?

I'm not sure what was there before, but after v0.9, dammif had the "unlimited search volume" which means that the search area starts with one single bead and grows from there by adding dummy atoms as needed. The idea is to avoid artifacts one may observe if Dmax was chosen too small. For example, let there be a spherical search volume and the particle touches on opposite sides of the search area so that one can see from one side to the other, or one specifies a cylindrical search volume although spherical would have been better - or vice versa. In any situation. one possibly would have needed some more beads to get better results. DAMMIF provides this by not limiting the search volume.

spark wrote
> I am declined to use D_Max for my search volume as I don't expect my particle to be bigger.

As others put it:
Jan L. A. van de Snepscheut wrote
> In theory, there is no difference between theory and practice. But, in practice, there is.

In other words, if your particle doesn't need to grow bigger than Dmax, then it won't :)

HTH

Daniel

P.S. also see "Franke, D. and Svergun, D.I. (2009) DAMMIF, a program for rapid ab-initio shape determination in small-angle scattering. J. Appl. Cryst., 42, 342-346. "

**Comment by franke-biosaxs** [comment]
On 2012-03-27 14:58:37, user spark wrote:

Dear SaxsMax and Franke,
thank you for your replies.

Franke, thank you for your detailed explanation why an unlimited search might be the better option.
Regarding a too small chosen Dmax, would it not be reasonable to provide for DAMMIN/F an error for Dmax? As far as I understand Dmax is not a hard parameter.

I read your new paper and it seemed with the new ATSAS version some things have changed, providing now an alternative route via DAMCLUST.

Personally for me, it gets more complicated to decide what is the best, probable model, rather simplifying this whole task.

In case I would like to follow the DAMAVER route as before, I just could evaluate the online results with DAMAVER as before, correct, please?

In that context I would also ask please for clarification of a good NSD value as there are several statements in the past published papers.

It is written in Kozin & Svergun (2001):
"For ideally superimposed similar objects, NSD tends to 0; it exceeds 1 if the objects systematically differ from one another."

On the other hand it is written in a paper by Koch & Svergun in 2002:
"A lower NSD corresponds to a better overlap and NSD of approximately 1 indicates a good agreement between two models at low resolution."

To use your quote, do theory and practice differ here as well? People accept models with NSD values of 1, saying those are good models, but actually this is not the whole truth as it is way off from 0?

Also when publishing the reconstructed model, do you publish the model serving as reference showing the lowest NSD value or maybe instead the avg.pdb-file?

Thank you very much for your asnwers.

Kind regards
Spark

**Comment by franke-biosaxs** [comment]
On 2012-04-16 04:44:10, user renje464 wrote:

Hi, I have read the specified papers however I am still a little confused regarding the DAMCLUST output and how to interpret the results. I used the online DAMMIF function so am I right in saying that DAMCLUST has also been applied to my data? I am unable to find a manual for DAMCLUST so was wondering how to interpret the results, specifically how do
DAMMIF-1-r differ form DAMMIF-1? and what would you use to determine which of the 20 runs is the best structure??
Thanks

**Comment by franke-biosaxs** [comment]
On 2013-04-11 23:27:55, user 123 wrote:

Any answers to renje464 questions?
It would be great to get some information about the different outputs of online DAMMIF. I couldn't find this described anywhere.

Thanks!

**Comment by franke-biosaxs** [comment]
On 2013-04-12 09:59:20, user AL wrote:


renje464 wrote
> I used the online DAMMIF function so am I right in saying that DAMCLUST has also been applied to my data?

Yes.

renje464 wrote
> I am unable to find a manual for DAMCLUST so was wondering how to interpret the results, specifically how do DAMMIF-1-r differ form DAMMIF-1?

[DAMCLUST manual](https://biosaxs-com.github.io/atsas/latest/manuals/damclust.html).
DAMMIF-1-r is the same model as DAMMIF-1 but [superimposed with the representative (most typical model) of the corresponding cluster](https://biosaxs-com.github.io/atsas/latest/manuals/damclust.html#output).


renje464 wrote
> and what would you use to determine which of the 20 runs is the best structure??

E.g. in case
spark wrote
> My pdb files were divided into groups, one composed of 3 pdb files and the other contains 15 files.

I'd call the representative of the second cluster (15 models) the "best" structure.
